editor/project_manager/project_list.cpp

```diff
@@ -984,7 +990,7 @@ void ProjectList::_create_project_item_control(int p_index) {
 	hb->set_tags(item.tags, this);
 	hb->set_unsupported_features(item.unsupported_features.duplicate());
 	hb->set_project_version(item.project_version);
-	hb->set_last_edited_info(!item.missing ? Time::get_singleton()->get_datetime_string_from_unix_time(item.last_edited, true) : TTR("Missing Date"));
+	hb->set_last_edited_info(!item.missing ? Time::get_singleton()->get_datetime_string_from_unix_time(item.last_edited, true) : TTRC("Missing Date"));
```

Comment:
The `yyyy-mm-dd HH:MM:SS` datetime string is untranslatable. I think we should disable auto translation for the label and keep this `TTR`.